src/ConfidentialLedger/examples/Get-AzConfidentialLedger.md

```diff
@@ -0,0 +1,20 @@
+### Example 1: List Confidential Ledgers
+```powershell
+PS C:\> Get-AzConfidentialLedger `
+  -SubscriptionId 00000000-0000-0000-0000-000000000000
+
+Location Name               
+eastus   testledger0
+eastus   testledger1
```

Comment:
It should follow the schema below :

```powershell 
PS C:\> {{ Add code here }}
{{ Add output here }}
```

{{ Add description here }}

Please add description for all examples